src/gfx_d3d12.zig

```diff
@@ -205,6 +219,7 @@ const PipelineHashMap = std.HashMap(IdLocal, PipelineInfo, IdLocalContext, 80);
 
 pub const D3D12State = struct {
     pub const num_buffered_frames = zd3d12.GraphicsContext.max_num_buffered_frames;
+    pub const max_num_lights: u32 = 1000;
```

Comment:
Similarly to my comment below, I tend to name things by "category" (not sure how to explain), but: `light_count_max`.